app/components/carousel/carousel.tsx

```diff
@@ -0,0 +1,95 @@
+'use client'
+import React from 'react'
+import Slider from 'react-slick'
+import { Box, IconButton, useBreakpointValue } from '@chakra-ui/react'
+import { LeftArrow, RightArrow } from '@/icons'
+
+type CarouselProps = {
+  urls: string[]
+}
+
+const settings = {
+  dots: true,
+  arrows: false,
+  fade: true,
+  infinite: true,
+  autoplay: false,
+  speed: 500,
+  autoplaySpeed: 5000,
+  slidesToShow: 1,
+  slidesToScroll: 1,
+}
+
+export const Carousel = ({ urls }: CarouselProps) => {
+  const [slider, setSlider] = React.useState<Slider | null>(null)
+
+  const top = useBreakpointValue({ base: '50%', md: '50%' })
+  const side = useBreakpointValue({ base: '-20px', md: '-30px' })
+
+  return (
+    <Box
+      position={'relative'}
+      width={{ base: '80%', md: '80%', lg: '100%' }}
+      height={{ base: '215px', md: '320px' }}
+      margin={'auto'}
+    >
+      <link
+        rel="stylesheet"
+        type="text/css"
+        href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css"
+      />
+      <link
+        rel="stylesheet"
+        type="text/css"
+        href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick-theme.min.css"
+      />
+      {/* Left Icon */}
+      <IconButton
```

Comment:
Could you fix the carousel icon buttons?

1. The place looks like incorrect. Not centre so I think side is -20px is correct.
<img width="855" alt="Screenshot 2023-12-04 at 3 45 52 PM" src="https://github.com/tabi-memo/frontend/assets/66394413/68fa109b-b8aa-4e91-9e43-f6595a201986">

2.  The icon is too big and the arrow is not centre of button
<img width="453" alt="Screenshot 2023-12-04 at 3 45 56 PM" src="https://github.com/tabi-memo/frontend/assets/66394413/b5d731be-66bd-4716-b0a8-31c94cb13fb5">

3. When it's dark mode, you can not see the below buttons 
<img width="497" alt="Screenshot 2023-12-04 at 3 47 17 PM" src="https://github.com/tabi-memo/frontend/assets/66394413/a6e1187d-ec63-4fd5-aa59-7d1ace32440f">